gnoland/website/views/faucet.html

```diff
@@ -14,14 +14,19 @@
 
 This is the Gno.land (test) faucet.
 
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+
+<!-- <form method="POST" action="http://localhost:5050/"> -->
 <form method="POST" action="http://gno.land:5050/">
   <label for="toaddr">GNO.land address (e.g. g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5):</label><br>
   <input type="text" id="toaddr" name="toaddr"/><br>
   <input type="submit" value="GO"/>
+  <div id="captcha" class="g-recaptcha" data-sitekey="{YOUR_SITE_KEY}"></span>
```

Comment:
can you make this configurable from a command line argument, i.e., by using the `html/template` package?

Reply:
and if the command line argument is not set, can you just not check for the captcha? this way, people can more easily run a faucet on a local testnet